generic/tests/monotonic_time.py

```diff
@@ -71,7 +71,7 @@ def test_Gtod(self):
         self._test(test_type='gtod', threshold=0)
 
     def test_Tsc(self):
```

Comment:
IMO, It is not convenient to define functions for each `test_type`.   It is better to drop it and just define the `test_type` we want to cover.  
But i am ok with the current change too, if update the function name to `test_tsc_lfence` to keep align with the `test_type`. Thanks!